ragna/deploy/_cli/core.py

```diff
@@ -171,3 +177,157 @@ def wait_for_api() -> None:
         if process is not None:
             process.kill()
             process.communicate()
+
+
+@corpus_app.command(help="Ingest some documents into a given corpus.")
+def ingest(
+    documents: list[Path],
+    corpus_name: Optional[str] = typer.Option(
+        None, help="Name of the corpus to ingest the documents into."
+    ),
+    config: ConfigOption = "./ragna.toml",  # type: ignore[assignment]
+    user: Optional[str] = typer.Option(
+        None, help="User to link the documents to in the ragna database."
+    ),
+    verbose: bool = typer.Option(
+        False, help="Print the documents that could not be ingested."
+    ),
+    ignore_log: bool = typer.Option(
```

Comment:
Short-term solution for checkpointing on files which have already been ingested. This option ignores the checkpoint logic.

Longer-term, I think this needs to be incorporated into the `ragna` database. Otherwise, it's very easy to add the same file to the vector database multiple times, which reduces performance (instead of returning 10 sources, you just get 10 copies of the same source), unless we have some logic to deal with this already.